### PR TITLE
Heroku draft

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

File: docs/hosting/server-setups/heroku.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Hosting n8n on Heroku
+
+This hosting guide shows you how to self-host n8n on Heroku. It uses:
+
+- [Docker Compose](https://docs.docker.com/compose/){:target="\_blank" .external-link} to create and define the application components and how they work together.
+- [Heroku's PostgreSQL service](https://devcenter.heroku.com/categories/heroku-postgres){:target="\_blank" .external-link} to host n8n's data storage.
+- [Heroku's CLI tool](https://devcenter.heroku.com/articles/heroku-cli){:target="\_blank" .external-link}.
```
Comment (StarfallProjects) [inline docs/hosting/server-setups/heroku.md:7]: Is this actually used?

Comment (ChrisChinchilla) [reply docs/hosting/server-setups/heroku.md:7]:
```suggestion
```

**Comment by ChrisChinchilla** [comment]
@StarfallProjects Changes made and approved

File: docs/hosting/server-setups/heroku.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Select **Deploy app**.
+
+After Heroku builds and deploys the app it provides links to **Manage App** or **View** the application.
+
+!!! note "Heroku and DNS"
+	Refer to the [Heroku documentation](https://devcenter.heroku.com/categories/networking-dns){:target="_blank" .external-link} to find out how to connect your domain to a Heroku application.
+
+## Changing the deployment template
+
+You can make changes to the deployment template by forking the repository and deploying from you fork.
```
Comment (StarfallProjects) [inline docs/hosting/server-setups/heroku.md:40]: We should link to the repo

File: docs/hosting/server-setups/heroku.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+This hosting guide shows you how to self-host n8n on Heroku. It uses:
+
+
+- [Docker Compose](https://docs.docker.com/compose/){:target="_blank" .external-link} to create and define the application components and how they work together.
+- [Heroku's PostgreSQL service](https://devcenter.heroku.com/categories/heroku-postgres){:target="_blank" .external-link} to host n8n's data storage.
+- A **Deploy to Heroku** button offering a one click, with minor configuration, deployment.
+
+
+## Use the deploy template to create a Heroku project
```
Comment (StarfallProjects) [inline docs/hosting/server-setups/heroku.md:11]:
```suggestion
## Use the deployment template to create a Heroku project
```

**Pull request merged**